cli/config/configuration-nemu.toml.in

```diff
@@ -123,6 +123,14 @@ virtio_fs_cache_size = @DEFVIRTIOFSCACHESIZE@
 #    Metadata, data, and pathname lookup are cached in guest and never expire.
 virtio_fs_cache = "@DEFVIRTIOFSCACHE@"
 
+# Extra args for virtiofsd daemon
+#
+# Format example:
+#   ["-o", "arg1=xxx,arg2", "-o", "hello world", "--arg3=yyy"]
+#
+# see `virtiofsd -h` for possible options.
+virtio_fs_extra_args = []
```

Comment:
I think you should replace `[]` with `@DEFVIRTIOFSEXTRAARGS@` to honour Makefile configuration .. :thinking:

Reply:
I overlooked that, thanks, done.